terraform/aks/variables.tf

```diff
@@ -30,6 +42,18 @@ variable "postgres_flexible_server_sku" { default = "B_Standard_B1ms" }
 variable "postgres_enable_high_availability" { default = false }
 variable "azure_enable_backup_storage" { default = true }
 
+variable "replicas" { default = 1 }
+variable "memory_max" { default = "1Gi" }
+
 locals {
   azure_credentials = try(jsondecode(var.azure_credentials), null)
+  kv_app_secrets    = yamldecode(data.azurerm_key_vault_secret.app_secrets.value)
```

Comment:
The application configuration module already extracts the YAML secrets and sets them as environment variables so I don't think you need this.